server/graphql/types/src/types/invoice_query.rs

```diff
@@ -95,19 +94,6 @@ impl InvoiceNode {
         self.invoice.other_party_id()
     }
 
-    pub async fn other_party_store(&self, ctx: &Context<'_>) -> Result<Option<StoreNode>> {
```

Comment:
We have other_party which has store so have removed. Also doesn't look like it's being used anywhere